### PR TITLE
Add nixd support

### PR DESCRIPTION
**Pull request by colonelpanic8**

### CODE REVIEW

File: clients/lsp-nix.el [hunk 1]
```diff
@@ -87,6 +87,27 @@
                   :synchronize-sections '("nil")
                   :server-id 'nix-nil))
 
+(defgroup lsp-nix-nixd nil
+  "LSP support for Nix, using nil."
```
Comment (psibi) [inline clients/lsp-nix.el:91]: Probably you meant `nixd` here ?

**Review by psibi — COMMENTED** [review]
LGTM, apart from a minor comment.

**Comment by paperdigits** [comment]
Can we change that comment please and merge this? I'd love to have nixd in emacs lsp-mode

**Comment by jcs090218** [comment]
Can you rebase this? Thanks!

Anyway, you might want to use #4089 as a reference. Changelog, `docs/lsp-clients.json`, etc.

**Comment by yyoncho** [comment]
Can you move this to a separate file?

**Comment by KarthikNayak** [comment]
@colonelpanic8 is this something you still want to pursue? I can take over if not. :)

**Comment by colonelpanic8** [comment]
Yeah go ahead!

**Comment by KarthikNayak** [comment]
@colonelpanic8 seems like it's already done 
https://github.com/emacs-lsp/lsp-mode/pull/4288

Thanks, @peterbecich.

Could one of the mods, close this issue? @psibi ?

**Comment by psibi** [comment]
@KarthikNayak Can you move it to a separate file as requested by Ivan in the thread ?

**Comment by psibi** [comment]
Ah, I see what you mean. It was in a different PR. Closing this.

**Pull request closed without merging**